src/SDK.ts

```diff
@@ -478,15 +478,14 @@ export default class SDK implements ISDK {
  * Fetches the reconnectable chats from omnichannel from the given user information in JWT token(claim name: sub).
  * @param reconnectableChatsParams Mandate parameters for get reconnectable chats.
  */
-  public async getReconnectAvailability(reconnectId: string): Promise<ReconnectAvailability | void> {
+  public async getReconnectAvailability(reconnectId: string, requestId: string): Promise<ReconnectAvailability | void> {
```

Comment:
Would that not be backward compatible? I'd use add `optionalParams = {}` in the signature instead as precaution